src/Paprika/Store/StorageFanOutPage.cs

```diff
@@ -102,6 +102,30 @@ public void Report(IReporter reporter, IPageResolver resolver, int level)
             }
         }
     }
+    public void Destroy(IBatchContext batch, in NibblePath prefix)
+    {
+        // Destroy the Id entry about it
+        Set(prefix, ReadOnlySpan<byte>.Empty, batch);
+
+        var map = new SlottedArray(Data.Data);
+        map.DeleteByPrefix(prefix);
+        var sliced = prefix.SliceFrom(ConsumedNibbles);
+        var index = GetIndex(sliced);
+        ref var addr = ref Data.Addresses[index];
+        Page child;
+        if (addr.IsNull)
+        {
+            //recycle for reuse
+            var indexOfPrefix = GetIndex(prefix);
```

Comment:
Why do anything while child is null? No need to do it.

Reply:
Corrected.